### ISSUE TITLE
Exit Status 1

### ISSUE THREAD

**Issue by alfonsogab** [malice-plugins/fsecure#1]
It seems that the scanning always return an error with exit status 1
"{"fsecure":{"infected":false,"result":"","results":{"fse":"","aquarius":""},"engine":"","database":"","updated":"","error":"exit status 1"}}"

**Comment by fareedfauzi** [comment]
> It seems that the scanning always return an error with exit status 1
> "{"fsecure":{"infected":false,"result":"","results":{"fse":"","aquarius":""},"engine":"","database":"","updated":"","error":"exit status 1"}}"

Yes, happened to me also. The bug seems like not will be solve by the malice since the issue opened last year haha.
